examples/jsm/loaders/OBJLoader.js

```diff
@@ -11,8 +12,7 @@ import {
 	MeshPhongMaterial,
 	Points,
 	PointsMaterial,
-	Vector3,
-	Color
+	Vector3
```

Comment:
This import rearrangement is because my editor sorts imports on save.

I can go through and do this for other files in the codebase if any the maintainers think that would be beneficial.

Reply:
The rearranged imports here are fine, but I don't think it's necessary to reorder imports in unchanged files – thanks!

Reply:
You're welcome! And thanks for clarifying.